lib/chartmogul/api_resource.rb

```diff
@@ -83,6 +83,7 @@ def self.build_connection
       Faraday.new(url: ChartMogul.api_base) do |faraday|
         faraday.use Faraday::Request::BasicAuthentication, ChartMogul.account_token, ChartMogul.secret_key
         faraday.use Faraday::Response::RaiseError
+        faraday.request :multipart
```

Comment:
this looks like you're making every request multipart?

Reply:
I think we can mix both (see: https://lostisland.github.io/faraday/middleware/multipart)
I will make sure that other requests aren't sent multipart. Maybe it is possible to avoid having a second connection.

Reply:
@pkopac 
For all other http requests fired by our Ruby Client, we do specify `req.headers['Content-Type'] = 'application/json'`. 
However: 
> The Multipart middleware converts a Faraday::Request#body Hash of key/value pairs into a multipart form request, but only under these conditions:
> 
>     - The request’s Content-Type is “multipart/form-data”
>     - Content-Type is unspecified, AND one of the values in the Body responds to #content_type.

Thus, csv files are sent as `multipart/form-data`. All other requests are being sent as `application/json`.

Double check:

I deleted all the fixtures files and  recorded new cassettes. Only new specs for csv are sent as `multipart`.